### ISSUE TITLE
[14.0.55] Pouvoir arrêter un précalcul "proprement"

### ISSUE THREAD

**Issue by Bonvallot** [Inist-CNRS/lodex#2063]
Sur le même principe que les enrichissements, pouvoir arrêter un précalcul en cours.
En effaçant les tâches de fond, il semble que le précalcul soit arrêté mais après avoir Dépublié impossible de Publier : instance ko

**Comment by touv** [comment]
Effectivement, il y a un ou plusieurs problèmes dans l’arrêt d'un pré-calcul.
Actuellement, Lodex ne peut pas techniquement arrêter un ws de précaclul, pour cela il est nécessaire de mettre à jour tous les webservices pour qu'ils utilisent une version ezs supérieure ou égale à la version 3.10 (voir avec @parmentf )

L’arrêt est donc local, partiel ou total en fonction du moment ou le calcul a été arrêté.

Saurais-tu dire à quelle étape, le calcul a été arrêté ?

**Comment by parmentf** [comment]
J'ai proposé de mettre à jour [l'image de base](https://github.com/Inist-CNRS/web-services/blob/ed4d850feeb7c6d587fe74522ba2ecd981ab267b/bases/ezs-python-server/Dockerfile#L12-L15) des services web lors du prochain *sprint*, donc pendant l'été.
C'est un travail délicat: il faudra mettre à jour aussi tous les services web (mais au minimum les services asynchrones).

**Comment by AnaelKremer** [comment]
closed by #3323
